apps/native-component-list/src/screens/ExpoApisScreen.tsx

```diff
@@ -69,7 +69,8 @@ const screens = [
   'Permissions',
   'Print',
   'Random',
-  'Recording',
+  'Recording (expo-av)',
+  'Recording (expo-audio)',
```

Comment:
It might be better to have just one main screen for a package instead of splitting it to many screens. If we need more screens for a package, I'd recommend doing it like we do with expo-image and now expo-modules-core too

Reply:
The apis are different so we can't reuse the same screen for both and we'll be keeping AV around for a while yet so I don't want to break it's example

Reply:
No no, I mean that we should have one main screen for Audio and Recording (with nested navigation) since they both are examples for the same package.

Reply:
Oh, sorry, I get you now. Will do 👍